tests/CacheCompat/CommonCache.Test.Common/CommandLineOptions.cs

```diff
@@ -25,10 +25,20 @@
 // 
 // ------------------------------------------------------------------------------
 
+using System;
 using CommandLine;
 
 namespace CommonCache.Test.Common
 {
+    [Flags]
```

Comment:
Nice, haven't seen one of those in ages.

Reply:
I grew up on C and C++.  I love my Flags Enums... ;)